app/controllers/lessons_controller.rb

```diff
@@ -3,6 +3,23 @@ def index
     @lessons = policy_scope(Lesson).order(created_at: :desc)
   end
 
+  def new
+    @lesson = Lesson.new
+    authorize @lesson
+  end
+
+  def create
+    @lesson = Lesson.new(lesson_params)
+    @lesson.user = current_user
+    if @lesson.save
+      authorize @lesson
```

Comment:
can perhaps refactor to put before @lesson.save

Reply:
Sounds good - tested and works - will push again